### PR TITLE
use framework  refactor code in test/e2e/common

### PR DESCRIPTION
**Pull request by tanjunchen**
**What type of PR is this?**
/kind cleanup
/release-note-none
/priority backlog
**What this PR does / why we need it**:
**Which issue(s) this PR fixes**: 
This makes e2e tests use the function under test/e2e/common.
Ref: #79686
**Special notes for your reviewer**:
/assign @oomichi 
**Does this PR introduce a user-facing change?**:
**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

### CODE REVIEW

File: test/e2e/common/pods.go [hunk 1]
```diff
@@ -27,7 +27,7 @@ import (
 
 	"golang.org/x/net/websocket"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```
Comment (oomichi) [inline test/e2e/common/pods.go]: ditto

File: test/e2e/common/pods.go [hunk 2]
```diff
@@ -829,15 +829,14 @@ var _ = framework.KubeDescribe("Pods", func() {
...
+		framework.ExpectEqual(podClient.PodIsReady(podName), false, "Expect pod's Ready condition to be false initially.")
 
 		ginkgo.By(fmt.Sprintf("patching pod status with condition %q to true", readinessGate1))
 		_, err := podClient.Patch(podName, types.StrategicMergePatchType, []byte(fmt.Sprintf(patchStatusFmt, readinessGate1, "True")), "status")
 		framework.ExpectNoError(err)
 		// Sleep for 10 seconds.
 		time.Sleep(maxReadyStatusUpdateTolerance)
-		gomega.Expect(podClient.PodIsReady(podName)).To(gomega.BeFalse(), "Expect pod's Ready condition to be false with only one condition in readinessGates equal to True")
-
+		framework.ExpectEqual(podClient.PodIsReady(podName), false, "Expect pod's Ready condition to be false with only one condition in readinessGates equal to True")
```
Comment (oomichi) [inline test/e2e/common/pods.go]: Please keep a blank line for the readability.

File: test/e2e/common/runtimeclass.go [hunk 3]
```diff
@@ -34,7 +34,6 @@ import (
 	utilpointer "k8s.io/utils/pointer"
 
 	"github.com/onsi/ginkgo"
-	"github.com/onsi/gomega"
```
Comment (oomichi) [inline test/e2e/common/runtimeclass.go]: Cool, ++

File: test/e2e/common/pods.go (hunk 2 above)
Comment (tanjunchen) [reply test/e2e/common/pods.go]: Sure

File: test/e2e/common/pods.go (hunk 1 above)
Comment (tanjunchen) [reply test/e2e/common/pods.go]: This is due to the plugin check mechanism of vscode.

**Comment by tanjunchen** [comment]
@oomichi done

**Review by oomichi — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**